### PR TITLE
Shooter Method

### LINKED ISSUE #26: Create Shooter command

### PR DESCRIPTION
**Pull request by MaramS09**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Shooter extends SubsystemBase {
+  /**
+   * Creates a new Shooter.
+   */
+  private CANSparkMax motor1 = new CANSparkMax(0, MotorType.kBrushless);
+  private CANSparkMax motor2 = new CANSparkMax(0, MotorType.kBrushless);
```
Comment (duffym22) [inline src/main/java/frc/robot/subsystems/Shooter.java]: Use CAN ID constants as declared in Constants.java

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Shooter extends SubsystemBase {
+  /**
+   * Creates a new Shooter.
+   */
+  private CANSparkMax shooter = new CANSparkMax(9, MotorType.kBrushless);
+  private Solenoid adjustmentSolenoid = new Solenoid(5);
+  private Solenoid rotationSolenoid = new Solenoid(6);
```
Comment (duffym22) [inline src/main/java/frc/robot/subsystems/Shooter.java]: Use constants defined in Constants.java

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  public Shooter() {
+
+  }
```
Comment (duffym22) [inline src/main/java/frc/robot/subsystems/Shooter.java:27]: Depending on what the design shows, may need to specify 2 motor controllers and for one to follow the other

**Review by duffym22 — APPROVED** [review]
Clean code, builds, will open new issue to add a 2nd motor controller and define in the constructor to follow primary motor controller.

**Review by mott-lab — APPROVED** [review]
Looks good!

**Pull request merged**